### DISCUSSION TITLE
How do I keep the Header, Footer and Side Bar locked in place?

### DISCUSSION THREAD

**Discussion by GlacierFox** [microsoft/fluentui-blazor#1824] [Q&A]
Hi, just wondering how the Header bar, Footer and Side Bar are locked in place on the flutter documentation site here:

https://www.fluentui-blazor.net/

I've added some basic code to the base template and I can't get all three components to stop scrolling with the rest of the content.

**Comment by vnbaaij** [comment]
Hi,

The source code for the demo site is part of this repo. It can be found in the examples folder. The live site uses the Client project (WebAssembly). When running locally you could choose to run the Server project. They both basically run the same code so most is placed in the Shared project.

**Reply by GlacierFox** [reply]
Ah I didn't know that, thank you @vnbaaij , I'll take a look at it!
